plugin/files/airconsole/runtime.js

```diff
@@ -374,6 +384,42 @@ cr.plugins_.AirConsole = function(runtime)
     ret.set_string(this.ac_uid);
   };
 
+  Exps.prototype.ControllerDeviceIDs = function (ret)
+  {
+    var arr = this.air_console.getControllerDeviceIds();
+
+    // Ok, let's create a fake C2 JSON string that is compatible with C2 array load function
+    var jsonStr = new Object();
+    jsonStr['c2array'] = true;
+    jsonStr['size'] = [arr.length, 1, 1];
+    var data = [];
+    for (var i in arr) {
+      data.push([[arr[i]]]);
+    }
+    jsonStr['data'] = data;
+
+    ret.set_string(JSON.stringify(jsonStr));
+  };
+
+  Exps.prototype.MasterControllerDeviceID = function (ret)
+  {
+    // getMasterControllerDeviceId can return undefined, so let's return 0 (always SCREEN) in that case
+    var id = (typeof this.air_console.getMasterControllerDeviceId());
+    ret.set_int((id !== 'number') ? 0 : id);
```

Comment:
Guess that needs to be checked C2 side with a AirConsole.MasterControllerDeviceId != 0. Returning undefined could mess up and trigger invisible JS errors for a lambda user